front/lib/api/assistant/configuration/browse.ts

```diff
@@ -0,0 +1,49 @@
+import type { BrowseConfigurationType, ModelId } from "@dust-tt/types";
+import { Op } from "sequelize";
+
+import { DEFAULT_BROWSE_ACTION_NAME } from "@app/lib/api/assistant/actions/names";
+import { AgentBrowseConfiguration } from "@app/lib/models/assistant/actions/browse";
+
+export async function fetchBrowseActionsConfigurations({
+  configurationIds,
+  variant,
+}: {
+  configurationIds: ModelId[];
+  variant: "light" | "full";
+}): Promise<Map<ModelId, BrowseConfigurationType[]>> {
+  if (variant !== "full") {
```

Comment:
TBH I feel it would be better to not pass variant to these functions, and just handle that in parent (i.e, not call the function at all if variant is not full)

But I can live with this

Reply:
Will give it some more love in a follow up PR.